package.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
add coverage publisher to support html and cobertura (used by Jenkins)
coverageReporter: {
      reporters: [
        { type: 'html', dir: 'coverage/html' },
        { type: 'cobertura', dir: 'coverage/cobertura' }
      ]
    }